zenoh/src/session.rs

```diff
@@ -1621,6 +1812,29 @@ impl SessionDeclarations for Arc<Session> {
             destination: Locality::default(),
         }
     }
+
+    /// Obtain a [`Liveliness`] struct tied to this Zenoh [`Session`].
+    ///
+    /// # Examples
+    /// ```
+    /// # async_std::task::block_on(async {
+    /// use zenoh::prelude::r#async::*;
+    ///
+    /// let session = zenoh::open(config::peer()).res().await.unwrap().into_arc();
+    /// let liveliness = session
+    ///     .liveliness()
+    ///     .declare_token("key/expression")
+    ///     .res()
+    ///     .await
+    ///     .unwrap();
+    /// # })
+    /// ```
+    #[zenoh_core::unstable]
+    fn liveliness(&self) -> Liveliness<'static> {
```

Comment:
Is there any plan to add some tests that verify the correct functioning of the liveness?
I believe these should become part of the CI.